src/content/en/updates/posts/2016/03/access-usb-devices-on-the-web.markdown

```diff
@@ -0,0 +1,295 @@
+`chrome://flags/#enable-experimental-web-platform-features`, enable the
+highlighted flag, restart Chrome and you should be good to go.
+
+<img style="width:723px; max-height:205px" src="/web/updates/images/2016-03-02-access-usb-devices-on-the-web/web-usb-flag.png" alt="Web USB Flag highlighted in chrome://flags"/>
+
+## Privacy & Security
+
+### Attacks against USB devices
+
+The WebUSB API does not even try to provide a way for any web page to
+connect to arbitrary USB devices. There are plenty of published attacks against
+USB devices that makes it unsafe to allow this.
+
+For this reason, a USB device can define a set of origins that are allowed to
+connect to it. This is similar to the
+[CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS)
+mechanism. In other words, **WebUSB devices are associated with a web origin**
+and can only be accessed from a page from the same origin.
+
+### HTTPS Only
+
+Because this experimental API is a powerful new feature added to the Web,
+Chrome aims to make it available only to [secure
+contexts](https://w3c.github.io/webappsec/specs/powerfulfeatures/#intro). This
+means you'll need to build with TLS in mind.
+
+During development you'll be able to interact with WebUSB through
+http://localhost by using some tools like the [Chrome Dev
+Editor](https://chrome.google.com/webstore/detail/pnoffddplpippgcfjdhbmhkofpnaalpg)
+or the handy `python -m SimpleHTTPServer`, but to deploy it on a site you'll
+need to have HTTPS setup on your server. I personally enjoy [GitHub
+Pages](https://pages.github.com) for demo purposes.
+To add HTTPS to your server you'll need to get a TLS certificate and set
+it up. Be sure to check out [Security with HTTPS
+article](/web/fundamentals/security/)
+for best practices there.
+
+### User Gesture Required
+
+As a security feature, getting access to connected USB devices with
+`navigator.usb.requestDevice` must be called via a user gesture
+like a touch or mouse click.
+
+## Let's start coding
+
+The WebUSB API relies heavily on JavaScript
+[Promises](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise).
+If you're not familiar with them, check out this great
+[Promises tutorial](http://www.html5rocks.com/en/tutorials/es6/promises/). One
+more thing, `() => {}` are simply ECMAScript 2015 [Arrow
+functions](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+-- they have a shorter syntax compared to function expressions and lexically
+bind the value of `this`.
+
+### Get Access to USB devices
+
+You can either prompt the user to select a single connected USB device using
+`navigator.usb.requestDevice` or call `navigator.usb.getDevices` to get a list
+of all connected USB devices the origin has access to.
+
+The `navigator.usb.requestDevice` function takes a mandatory JavaScript object
+that defines `filters`. These filters are used to match any USB device with the
```

Comment:
Is this a good point to maybe link to the web bluetooth work? How the APIs are very similar etc etc?